pages/home.css

```diff
@@ -0,0 +1,106 @@
+.note-container {
+  margin: 2rem 5rem;
+}
+.wrapper {
+  display: flex;
+  padding: 2rem 0rem;
+  justify-content: space-between;
+}
+aside {
```

Comment:
dont use elements directly make a class and then apply CSS

Reply:
okay I will change this .